lib/index.js

```diff
@@ -0,0 +1,120 @@
+const fs = require('fs');
+const {
+  fetchLabels,
+  createLabels,
+  deleteLabels,
+  updateLabels,
+} = require('./api');
+
+async function labelControl() {
+  const publicLabels = await fetchLabels();
+  const data = fs.readFileSync('./label-config.json', { encoding: 'utf8' });
```

Comment:
```suggestion
  const data = fs.readFileSync(path.join(process.cwd(), './label-config.json'), { encoding: 'utf8' });
```

We need to write the file in the correct location (which is not inside this package)

Reply:
bca948f